common/persistence/utils/clusterMetadata.go

```diff
@@ -18,7 +18,9 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package persistence
+package utils
```

Comment:
nit: I kind of think having persistence in the package and import name is more informative than just `utils`, which is not a recommended package name (https://blog.golang.org/package-names#TOC_5.)

Reply:
yeah good point. I dislike that too. I moved the ClusterMetadata to `cluster` package, and renamed this to `persistence-util`